packages/react-ui-components/src/Badge/badge.tsx

```diff
@@ -0,0 +1,41 @@
+import mergeClassNames from 'classnames';
+import React, {PureComponent} from 'react';
+
+interface IBadgeTheme {
```

Comment:
Official TS guide doesn't recommend prefixing interfaces with I. Should we?

Reply:
Well, that's totally up to us.
I'm completely open minded here, no opinion at all.

Reply:
Yeah, makes sense